src/alt.js

```diff
@@ -77,7 +99,8 @@ class AltStore {
         )
       }
       if (model[LISTENERS][payload.action]) {
-        this[SET_STATE] = false
+        this[IS_DISPATCHING] = true
```

Comment:
Good question. Because someone may have registered themselves to the dispatcher or are using a lifecycle hook, in that case if you use setState it should still emit a change.

Example:

```js
function FooStore() {
  this.state = { calls: 0 }
  this.alt.dispatcher.register(function (payload) {
    this.setState({ calls: this.state.calls + 1 })
  });
}
```